src/Components/ApiFetch/ApodApiFetch.tsx

```diff
@@ -0,0 +1,24 @@
+﻿import React from "react";
+
+
+
+export function parseApiData() {
```

Comment:
Ideally we would use TypeScript and specify what `type` of object is returned by this method.
That will make it nice and easy to use it, as we will know what it returns.